build/media_source/system/js/fields/calendar-locales/date/gregorian/date-helper.es5.js

```diff
@@ -337,10 +371,10 @@
 			ir = 12;
 		var min = this.getMinutes();
 		var sec = this.getSeconds();
-		s["%a"] = JoomlaCalLocale.shortDays[w];                                                     // abbreviated weekday name
-		s["%A"] = JoomlaCalLocale.days[w];                                                          // full weekday name
-		s["%b"] = JoomlaCalLocale.shortMonths[m];                                                   // abbreviated month name
-		s["%B"] = JoomlaCalLocale.months[m];                                                        // full month name
+		s["%a"] = localStrings.shortDays[w];                                                     // abbreviated weekday name
+		s["%A"] = localStrings.days[w];                                                          // full weekday name
+		s["%b"] = localStrings.shortMonths[m];                                                   // abbreviated month name
+		s["%B"] = localStrings.months[m];                                                        // full month name
```

Comment:
```suggestion
		s["%a"] = localStrings.shortDays[w];                                                        // abbreviated weekday name
		s["%A"] = localStrings.days[w];                                                             // full weekday name
		s["%b"] = localStrings.shortMonths[m];                                                      // abbreviated month name
		s["%B"] = localStrings.months[m];                                                           // full month name
```
The code style change here somehow disappeared ... please redo it so the "//" are aligned with those below.